tests/parsers/beckman_coulter_biomek/testdata/example_01.json

```diff
@@ -0,0 +1,184 @@
+{
+    "$asm.manifest": "http://purl.allotrope.org/manifests/light-obscuration/BENCHLING/2023/12/light-obscuration.manifest",
+    "liquid handler aggregate document": {
+        "liquid handler document": [
+            {
+                "measurement aggregate document": {
+                    "measurement document": [
+                        {
+                            "device control aggregate document": {
+                                "device control document": [
+                                    {
+                                        "device type": "liquid handler",
+                                        "custom information document": {
+                                            "probe": "1",
+                                            "source pod": "Pod1",
+                                            "destination pod": "Pod1",
+                                            "source labware name": "Inducer",
+                                            "destination labware name": "GRP1",
+                                            "source liquid handling technique": "Custom",
+                                            "destination liquid handling technique": "Custom"
+                                        }
+                                    }
+                                ]
+                            },
+                            "sample document": {
+                                "sample identifier": "N/A",
+                                "source location identifier": "TL14",
+                                "destination location identifier": "TL15",
+                                "source well plate identifier": "96WP001",
+                                "destination well plate identifier": "96WP002",
+                                "source well location identifier": "1",
+                                "destination well location identifier": "1"
+                            },
+                            "measurement identifier": "BECKMAN_COULTER_BIOMEK_TEST_ID_0",
+                            "measurement time": "2024-09-04T14:16:00+00:00",
+                            "aspiration volume": {
+                                "value": 100.0,
+                                "unit": "μL"
+                            },
+                            "transfer volume": {
+                                "value": 98.0,
+                                "unit": "μL"
+                            }
+                        },
+                        {
+                            "device control aggregate document": {
+                                "device control document": [
+                                    {
+                                        "device type": "liquid handler",
+                                        "custom information document": {
+                                            "probe": "2",
+                                            "source pod": "Pod1",
```

Comment:
@nathan-stender, one note on the Pod. It should be the same for aspiration/dispense. May be redundant to specify source/destination.